### PR TITLE
Regenerate with latest gir and gir-files

### PR DESCRIPTION
**Pull request by EPashkin**
cc @GuillaumeGomez, @sdroege

### CODE REVIEW

File: atk-sys/tests/layout.c [hunk 1]
```diff
@@ -1,5 +1,5 @@
-// This file was generated by gir (https://github.com/gtk-rs/gir @ 5ec9961)
-// from gir-files (https://github.com/gtk-rs/gir-files @ 3fde76b)
+// This file was generated by gir (https://github.com/gtk-rs/gir @ c530bb9)
+// from gir-files (https://github.com/gtk-rs/gir-files @ a788da4)
```
Comment (sdroege) [inline atk-sys/tests/layout.c]: Can we switch to a single version file?

File: glib-sys/src/lib.rs [hunk 2]
```diff
@@ -3658,37 +3677,37 @@ extern "C" {
...
+    pub fn g_file_open_tmp_utf8(tmpl: *const c_char, name_used: *mut *mut c_char, error: *mut *mut GError) -> c_int;
+    pub fn g_file_open_tmp(tmpl: *const c_char, name_used: *mut *mut c_char, error: *mut *mut GError) -> c_int;
+    pub fn g_file_read_link(filename: *const c_char, error: *mut *mut GError) -> *mut c_char;
+    pub fn g_file_set_contents(filename: *const c_char, contents: *mut u8, length: ssize_t, error: *mut *mut GError) -> gboolean;
     #[cfg(any(windows, feature = "dox"))]
-    pub fn g_file_test_utf8(filename: *mut c_char, test: GFileTest) -> gboolean;
-    pub fn g_file_test(filename: *mut c_char, test: GFileTest) -> gboolean;
-    pub fn g_filename_display_basename(filename: *mut c_char) -> *mut c_char;
-    pub fn g_filename_display_name(filename: *mut c_char) -> *mut c_char;
+    pub fn g_file_test_utf8(filename: *const c_char, test: GFileTest) -> gboolean;
```
Comment (sdroege) [inline glib-sys/src/lib.rs]: Why are these all changing?

**Comment by sdroege** [comment]
Generally looks ok. Before regenerating any of the non-sys crates we need to fix things for the filename annotation disaster

File: atk-sys/tests/layout.c (hunk 1 above)
Comment (EPashkin) [reply atk-sys/tests/layout.c]: Also it will be less effect as each sys need contain his `version.txt`

File: glib-sys/src/lib.rs (hunk 2 above)
Comment (EPashkin) [reply glib-sys/src/lib.rs]: Because it has const now: `<type name="filename" c:type="const gchar*"/>`

**Comment by EPashkin** [comment]
As I remember only glib and gio affected by filename.

**Comment by EPashkin** [comment]
..and I working on [it](https://github.com/gtk-rs/glib/pull/330) in slow pace.

**Comment by GuillaumeGomez** [comment]
Looks good to me as well. However, I'm a bit scared to merge for the moment because of @sdroege's concern.

**Comment by EPashkin** [comment]
@GuillaumeGomez As @sdroege says it affect only non-sys crates cas all these types is "c_char*"

**Comment by GuillaumeGomez** [comment]
Yes but it also means that if one of the non-sys crates updates its dependencies, it'll break, right?

**Comment by EPashkin** [comment]
Why?

**Comment by EPashkin** [comment]
I don't see reason for breaking. Only some functions in glib, gio now accept `PathBuf` instead `String` in non-sys and its can't affect other crates.

**Comment by GuillaumeGomez** [comment]
Ok then I misunderstood, my bad. If @sdroege is ok with merging it, then I'll just go along.

**Comment by sdroege** [comment]
Sure

**Comment by GuillaumeGomez** [comment]
Thanks @EPashkin!

**Pull request merged**